### DISCUSSION TITLE
I created a date picker component, how do I share it so it can be included in Geist?

### DISCUSSION THREAD

**Discussion by robertsamarji** [zeit-ui/react#690] [Show and tell]
Hey there, I created a date picker component that closely resembles the one used in Vercel's "Usage" section

 I've built it by composing a number of existing components and adding a good number of CSS overrides. I'm planning on creating a date range picker too with the quick presets as seen in Vercel.

Is there a way for me to share it so it can be included in the standard components list so people don't have to rebuild it?

Screenshot of (almost) finished component:

<img width="359" alt="Screenshot 2022-01-25 at 11 54 21" src="https://user-images.githubusercontent.com/26310556/150964409-6e7b4ee4-44f2-49b9-ae45-d642fdf676e3.png">

**Comment by unix** [comment]
Hello @robertsamarji , thanks for your sharing. From the screenshot, the components are really good. I've been thinking about geist not having too many components to avoid distracting the maintainer too much.
You can still create a Pull Request, which will show people how you built the component.

**Comment by robertsamarji** [comment]
Awesome, I'll do that.

**Comment by diegoulloao** [comment]
what about this?

**Comment by Aetrius** [comment]
@robertsamarji Did you make a pull request?
